MaxText/configs/quantization/mp_scale.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Add doc book or reference to AQT example/code.

Reply:
Added more comments in config file. Also an example of how to use it is here: https://github.com/google/maxtext/blob/74c9d6bb01e5437a340c36131a420255b6a7aae9/end_to_end/tpu/test_decode_save_quantized_ckpt.sh#L69